latch_cli/snakemake/workflow.py

```diff
@@ -84,6 +85,10 @@
 SnakemakeInputVal: TypeAlias = snakemake.io._IOFile
 
 
+def _is_dir(x: SnakemakeInputVal) -> bool:
+    return os.path.isdir(x)
```

Comment:
kinda odd to have a one line function here instead of using `os.path.isdir` in place of the function calls.

Reply:
True, I initially had it as 
`return os.path.isdir(x) or x.is_directory`
but realized that idk what x.is_directory even means ( I think its some Snakemake specific construct that seems to return false when x is a directory so remove that check. 

will inline this

Reply:
sgsg